package-system/AWSGameLiftServerSDK/build-linux.sh

```diff
@@ -0,0 +1,222 @@
+
+elif [ "${TARGET_ARCH}" = "aarch64" ] 
+then
+    echo "Processing Docker for aarch64"
+
+    DOCKER_INPUT_ARCHITECTURE=arm64v8
+    TARGET_DOCKER_PLATFORM_ARG=linux/arm64/v8
+    DOCKER_BUILD_ARG=3
+else
+    echo "Unsupported architecture ${TARGET_ARCH}"
+    exit 1
+fi
+
+# Prepare to build on both Ubuntu 20.04 and Ubuntu 22.04 based docker images
+
+mkdir -p ${TEMP_FOLDER}
+cp -f ${DOCKER_BUILD_SCRIPT} ${TEMP_FOLDER}/
+
+# Args
+# $1 : Ubuntu version
+# $2 : Include
+# $3 : Docker run platform
+
+function execute_docker() {
+
+    # Determine the openssl version based on the ubuntu version (20.04/OpenSSL 1.1.1.x vs 22.04/OpenSSL 3.x)
+    if [ $1 = "20.04" ]
+    then
+        echo "Preparing for OpenSSL 1.1.1.x version"
+        BIN_SUBFOLDER_NAME=openssl-1
+    elif [ $1 = "22.04" ]
+    then
+        echo "Preparing for OpenSSL 1.1.1.x version"
+        BIN_SUBFOLDER_NAME=openssl-3
+    else
+        echo "Unsupported base build image ubuntu version ${1}"
+        exit 1
+    fi
+
+    # Build the Docker Image 
+    DOCKER_IMAGE_NAME=${DOCKER_IMAGE_NAME_BASE}_focal_${DOCKER_INPUT_ARCHITECTURE}_3p
+
+    echo "Building the docker build script for ${DOCKER_IMAGE_NAME_BASE} on ${DOCKER_INPUT_ARCHITECTURE} for Ubuntu $1"
+    echo ""
+    echo docker build --build-arg INPUT_DOCKER_BUILD_SCRIPT=${DOCKER_BUILD_SCRIPT} \
+                 --build-arg INPUT_ARCHITECTURE=${DOCKER_INPUT_ARCHITECTURE} \
+                 --build-arg INPUT_IMAGE=ubuntu:${1} \
+                 -f Dockerfile -t ${DOCKER_IMAGE_NAME}:latest temp 
+    docker build --build-arg INPUT_DOCKER_BUILD_SCRIPT=${DOCKER_BUILD_SCRIPT} \
+                 --build-arg INPUT_ARCHITECTURE=${DOCKER_INPUT_ARCHITECTURE} \
+                 --build-arg INPUT_IMAGE=ubuntu:${1} \
+                 -f Dockerfile -t ${DOCKER_IMAGE_NAME}:latest temp 
+    if [ $? -ne 0 ]
+    then
+        echo "Error occurred creating Docker image ${DOCKER_IMAGE_NAME}:latest." 
+        exit 1
+    fi
+
+    # Run the build script in the docker image
+    echo "Running build script in the docker image ${DOCKER_IMAGE_NAME}"
+    echo ""
+    echo docker run --platform ${TARGET_DOCKER_PLATFORM_ARG} \
+                   -it --tty \
+                   ${DOCKER_IMAGE_NAME}:latest /data/workspace/${DOCKER_BUILD_SCRIPT} ${SDK_SRC_SUBPATH}
+    docker run --platform ${TARGET_DOCKER_PLATFORM_ARG} \
+               --tty \
+               ${DOCKER_IMAGE_NAME}:latest /data/workspace/${DOCKER_BUILD_SCRIPT} ${SDK_SRC_SUBPATH}
+    if [ $? -ne 0 ]
+    then
+        echo Failed to build from docker image ${DOCKER_IMAGE_NAME}:latest
+        echo "To log into and troubleshoot the docker container, run the following command:"
+        echo ""
+        echo "docker run --platform ${TARGET_DOCKER_PLATFORM_ARG} -it --tty ${DOCKER_IMAGE_NAME}:latest"
+        echo ""
+        exit 1
+    fi
+
+    echo "Build Complete"
+
+    echo "docker run --platform ${TARGET_DOCKER_PLATFORM_ARG} -it --tty ${DOCKER_IMAGE_NAME}:latest"
```

Comment:
```suggestion        
    echo "To log into the docker container, run the following command:"
    echo ""
    echo "docker run --platform ${TARGET_DOCKER_PLATFORM_ARG} -it --tty ${DOCKER_IMAGE_NAME}:latest"
```

Reply:
Actually, it should be removed. The lines above this print this message if an error occurs during the docker run